apps/api/src/app/workflows-v2/util/jsonToSchema.ts

```diff
@@ -67,4 +67,55 @@ function determineSchemaType(value: unknown): JSONSchemaDto {
   }
 }
 
-export { generateJsonSchema, JSONSchemaDto };
+function isMatchingJsonSchema(schema: JSONSchemaDefinition, obj?: Record<string, unknown> | null): boolean {
```

Comment:
The helper function that checks if the control values are matching with all schema required fields